server/parsec/cli/run.py

```diff
@@ -386,6 +386,42 @@ def handle_parse_result(
     show_envvar=True,
     help="SSL certificate file. This setting enables serving Parsec over SSL.",
 )
+@click.option(
+    "--ssl-ciphers",
+    type=str,
+    envvar="PARSEC_SSL_CIPHERS",
+    show_envvar=True,
+    callback=lambda ctx, param, value: value.split(","),
+    # List of recommended cipher suites by ANSSI
+    # https://cyber.gouv.fr/sites/default/files/2017/07/anssi-guide-recommandations_de_securite_relatives_a_tls-v1.2.pdf
+    default=",".join(
+        [
+            # We need to translate the cipher names to be compatible with openssl naming,
+            # We use https://docs.openssl.org/3.5/man1/openssl-ciphers/#cipher-suite-names for that.
+            # For TLSv1.3, openssl uses the same naming so no translation.
+            # https://docs.openssl.org/3.5/man1/openssl-ciphers/#tls-v13-cipher-suites
+            "TLS_AES_256_GCM_SHA384",
+            "TLS_AES_128_GCM_SHA256",
+            "TLS_AES_128_CCM_SHA256",
+            "TLS_CHACHA20_POLY1305_SHA256",
+            # For TLSv1.2
+            # https://docs.openssl.org/3.5/man1/openssl-ciphers/#tls-v12-cipher-suites
+            # using ECDSA
+            "ECDHE-ECDSA-AES256-GCM-SHA384",  # TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+            "ECDHE-ECDSA-AES128-GCM-SHA256",  # TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+            "ECDHE-ECDSA-AES256-CCM",  # TLS_ECDHE_ECDSA_WITH_AES_256_CCM
+            "ECDHE-ECDSA-AES128-CCM",  # TLS_ECDHE_ECDSA_WITH_AES_128_CCM
+            # https://docs.openssl.org/3.5/man1/openssl-ciphers/#chacha20-poly1305-cipher-suites-extending-tls-v12
+            "ECDHE-ECDSA-CHACHA20-POLY1305",  # TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256
+            # using RSA
+            "ECDHE-RSA-AES256-GCM-SHA384",  # TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+            "ECDHE-RSA-AES128-GCM-SHA256",  # TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+            # https://docs.openssl.org/3.5/man1/openssl-ciphers/#chacha20-poly1305-cipher-suites-extending-tls-v12
+            "ECDHE-RSA-CHACHA20-POLY1305",  # TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+        ]
+    ),
+    help="Comma separated of TLS cipher suite to use",
```

Comment:
Should improve the help with something similar than https://docs.gunicorn.org/en/stable/settings.html#ciphers